snapcraft/models/project.py

```diff
@@ -572,7 +572,7 @@ class Component(models.CraftBaseModel):
 
     summary: SummaryStr
     description: str
-    type: Literal["test"]
+    type: Literal["test", "kernel-modules", "standard"]
```

Comment:
This might be a better question for @alfonsosanchezbeato:

What is "standard" here? I understand `test` and `kernel-modules` and could understand things like `translations` or `locales`, but I don't understand what this is.

Reply:
From [MM](https://chat.canonical.com/canonical/pl/nntxrmbhx7n59fpsex4bftcp9w):

> it is the type that components without special behavior will use (that is, components that are just mounted and made available to the owner snap)